tests/core/test_memory.py

```diff
@@ -0,0 +1,52 @@
+import torch
+import torch.nn as nn
+
+from pytorch_lightning import Trainer, LightningModule
+from pytorch_lightning.core.memory import ModelSummary
+
+
+# TODO:
+# Empty LightningModule (no layers)
```

Comment:
you mean dd test parametrization?

Reply:
yeah something like this. just some WIP todo for myself. 
just want to make sure my refactor does not break anything since there were no tests before about model sumamry